packages/dashboard/src/customization/widgets/utils/getAggregationResolution.ts

```diff
@@ -0,0 +1,32 @@
+import { AggregateType } from '@aws-sdk/client-iotsitewise';
```

Comment:
just calling out the files that matter!
all these defaults are listed in the 5Q doc for aggregations+resolutions